gluten-ut/spark32/src/test/scala/org/apache/spark/sql/execution/joins/GlutenBroadcastJoinSuite.scala

```diff
@@ -19,7 +19,7 @@ package org.apache.spark.sql.execution.joins
 
 import io.glutenproject.backendsapi.BackendsApiManager
 import io.glutenproject.GlutenConfig
-import io.glutenproject.execution.{BroadcastHashJoinExecTransformer, GlutenColumnarToRowExecBase, WholeStageTransformerExec}
+import io.glutenproject.execution.{BroadcastHashJoinExecTransformer, ColumnarToRowExecBase, WholeStageTransformerExec}
```

Comment:
Not an issue on this line, but just recognized `WholeStageTransformerExec` can be named `WholeStageExecTransformer` or `WholeStageTransformer` for consistency.

Reply:
Rename `WholeStageTransformerExec` to `WholeStageTransformer`.